### PR TITLE
add washington state university to harvesters

### PR DESCRIPTION
**Pull request by kms6bn**
added harvester, test, and favicon for washington state university.

### CODE REVIEW

File: scrapi/harvesters/wash_state_u.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+'''
+from __future__ import unicode_literals
+
+from scrapi.base import OAIHarvester
+
+
+class WashuHarvester(OAIHarvester):
+    short_name = 'wash_state_u'
+    long_name = 'Washington State University Research Exchange'
+    url = 'http://research.wsulibs.wsu.edu:8080'
```
Comment (fabianvf) [inline scrapi/harvesters/wash_state_u.py]: This url is for people, I think it should be http://research.wsulibs.wsu.edu/xmlui/

**Comment by kms6bn** [comment]
Hm - ok, I'll look into it

**Comment by kms6bn** [comment]
Can you clarify? To me, it looks like its working and pulling articles (according to what it pulled into the test and my localhost)

**Comment by kms6bn** [comment]
Cool - let me know if you need me to change anything else!

**Comment by fabianvf** [comment]
:+1:

**Pull request merged**